### PR TITLE
Identify lcz

### PR DESCRIPTION
**Pull request by j3r3m1**
Still a problem to resolve with impervious fractions being to 0 for the MEDIAN method used for averaging.
Moreover, we need to separate more clearly the built LCZ from the natural ones (for example using the building_fraction).

### CODE REVIEW

**Comment by j3r3m1** [comment]
I have an idea to deal with the problem of separation between built LCZ and natural LCZ but I think that first we should identify whether this mistake happens frequently on real datasets.

File: geoindicators/src/main/resources/org/orbisgis/LCZ_classes.csv [hunk 1]
```diff
@@ -0,0 +1,18 @@
+name,sky_view_factor_inf,sky_view_factor_sup,aspect_ratio_inf,aspect_ratio_sup,building_surface_fraction_inf,building_surface_fraction_sup,impervious_surface_fraction_inf,impervious_surface_fraction_sup,pervious_surface_fraction_inf,pervious_surface_fraction_sup,height_of_roughness_elements_inf,height_of_roughness_elements_sup,terrain_roughness_class_inf,terrain_roughness_class_sup
```
Comment (ebocher) [inline geoindicators/src/main/resources/org/orbisgis/LCZ_classes.csv]: Is this file necessary ?

Comment (j3r3m1) [reply geoindicators/src/main/resources/org/orbisgis/LCZ_classes.csv]: not anymore we can remove it

**Comment by ebocher** [comment]
Check failed
